src/x11.rs

```diff
@@ -39,15 +39,15 @@ impl X11Impl {
     pub unsafe fn from_xlib(
         window_handle: XlibWindowHandle,
         display_handle: XlibDisplayHandle,
-    ) -> Result<Self, SwBufError> {
+    ) -> Result<Self, SoftBufferError> {
         // TODO: We should cache the shared libraries.
 
         // Try to open the XlibXCB shared library.
-        let lib_xcb = Xlib_xcb::open().swbuf_err("Failed to open XlibXCB shared library")?;
+        let lib_xcb = Xlib_xcb::open().softbuffer_err("Failed to open XlibXCB shared library")?;
```

Comment:
I find the `softbuffer_err` name a bit too verbose for my liking, do you mind changing it back to what it was before? This is just a convenience function that isn't publicly exposed.